### PR TITLE
Hunt teams

### PR DESCRIPTION
**Pull request by Tmee**

### CODE REVIEW

**Comment by trayo** [comment]
Nice! :rocket:

**Comment by Tmee** [comment]
struggling today

File: app/models/hunt.rb [hunk 1]
```diff
@@ -6,4 +6,20 @@ def locations
...
+      teams.create(team_number: i,
+                  path: generate_team_path)
+    end
+  end
+
+
+  private
+
+  def generate_team_path
+    Random.rand(1..2) == 1 ? reversed_path = master_path.reverse : reversed_path = master_path
```
Comment (trayo) [inline app/models/hunt.rb]: Using `Random` isn't necessary in this case, you can say just `rand`.

Comment (trayo) [inline app/models/hunt.rb]: Also, you can use `reversed_path =` at the beginning and it would eliminate the duplication in the middle of the ternary.

**Comment by Tmee** [comment]
do it

**Comment by dalexj** [comment]
merge this please

**Comment by trayo** [comment]
All is well!

**Pull request merged**